src/helperFunctions/commafy.ts

```diff
@@ -0,0 +1,11 @@
+// https://stackoverflow.com/questions/6784894/add-commas-or-spaces-to-group-every-three-digits
+export default function commafy(num: number) {
```

Comment:
I think it might be better to write a description here how does the function works

Reply:
Added function description.